aries-site/src/examples/components/tag/CreateNameValuePair.js

```diff
@@ -0,0 +1,241 @@
+import React, { useContext, useEffect, useState } from 'react';
+import {
+  Box,
+  Button,
+  Heading,
+  Grid,
+  Form,
+  FormField,
+  Select,
+  Tag,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { tagData } from './data';
+
+// the prefix name of the Create option entry
+const prefix = 'Create';
+
+const defaultNameOptions = [];
+const defaultValueOptions = [];
+const defaultName = '';
+const defaultValue = '';
+const defaultTags = [];
+
+Object.entries(tagData).forEach(([, value]) => {
+  defaultNameOptions.push({
+    label: value.displayName,
+    values: value.options,
+  });
```

Comment:
Per the user-generated aspect of this, we likely would not have a display name and thus this can simply push the key as a string.